### ISSUE TITLE
tutorials/your-first-mac-aspnet.md "Your First ASP.NET Core Application on a Mac Using Visual Studio Code"

### ISSUE THREAD

**Issue by danroth27** [aspnet/Docs#1842]
Add content to the Your First App on Mac tutorial to discuss the various artifacts in the generated project
Right now most of the content in the [Your First App on Mac tutorial](https://docs.asp.net/en/latest/tutorials/your-first-mac-aspnet.html) talks about Yeoman, Visual Studio Code and Azure. We should add content that actually goes through what is in the generate project.

**Comment by Rick-Anderson** [comment]
- [ ] First occurrence of Visual Studio Code in the article append (VSC), then refer to it as VSC. Visual Studio Code gets tedious.

**Comment by Rick-Anderson** [comment]
@spboyer  can you take a look at the Live Fyre questions. Folks can't get this to work.

**Comment by spboyer** [comment]
@Rick-Anderson addressed.  The yeoman generator is using a previous version of the sdk.  The fix is 

add a file to the root of the project called "global.json" with the following content

```
{
  "projects": [ "." ],
  "sdk": {
    "version": "1.0.0-preview2-1-003177"
  }
}
```

- if the source of the project is in the root then set the "projects" value to ".", otherwise set it to where ever it is. i.e. "src"
- set the version to the value of the dotnet version "1.0.0-preview2-1-003177", this is the version that yo aspnet is currently generating.

**Comment by FabianGosebrink** [comment]
Hey,

I would like to help concerning an article which could be called like "Your First ASP.NET Core Application using the dotnet CLI Visual Studio Code"? Would this be helpful in any way? Would do it on a windows system. This issue here seems technically close so I did not create a new one.

Could I help you out with that?

**Comment by danroth27** [comment]
@FabianGosebrink Thanks for offering to help with the ASP.NET Core docs! I think the intent is that the existing Mac+VSCode doc should already cover what you are offering to write. We will be updating this article to use the latest CLI commands including using `dotnet net` instead of Yeoman shortly.

We will also need to look at adding an article that covers how to get started with ASP.NET Core and Visual Studio for Mac. Filed https://github.com/aspnet/Docs/issues/2767 to track.

**Comment by spboyer** [comment]
Closed via #2831
